wallet/loader.go

```diff
@@ -86,7 +88,7 @@ func NewLoader(chainParams *chaincfg.Params, dbDirPath string,
 // to store the wallet. Given that the external DB may be shared an additional
 // function is also passed which will override Loader.WalletExists().
 func NewLoaderWithDB(chainParams *chaincfg.Params, recoveryWindow uint32,
-	db walletdb.DB, walletExists func() (bool, error)) (*Loader, error) {
+	db walletdb.DB, syncLoopInterval time.Duration, walletExists func() (bool, error)) (*Loader, error) {
```

Comment:
Would be nice to adhere to the 80 character line length limit (using 8 spaces for tabs) more or less observed throughout the codebase.